functions/compare-nr/1.0/index.js

```diff
@@ -0,0 +1,20 @@
+const compareNumber = async ({ leftValue, rightValue, comparator }) => {
+    switch (comparator) {
+        case 'eq':
+            return { result: leftValue === rightValue }
+        case 'neq':
+            return { result: leftValue !== rightValue }
+        case 'lower':
+            return { result: leftValue < rightValue }
+        case 'lowerEq':
+            return { result: leftValue <= rightValue }
+        case 'higher':
+            return { result: leftValue > rightValue }
+        case 'higherEq':
+            return { result: leftValue >= rightValue}
+        default:
+            return { result: false }
+    }
```

Comment:
Perhaps  we could move the `return` to the end of the function and declare a result variable. Something like
```
let result;

switch (comparator) {
  case 'eq':
    result = leftValue === rightValue;
  ...
  default:
    result = false;
}

return {
  result,
};
```